user_guide_src/source/outgoing/view_parser/023.php

```diff
@@ -1,3 +1,3 @@
 <?php
 
-return $parser->render('myview');
+return $parser->renderString($template);
```

Comment:
`$template` is not defined. It seems better to show working code.

Reply:
You are correct.